.github/workflows/ci.yaml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
Aren't pull requests enough since master branch is protected?

Reply:
Not necessarily. If the PR's branch isn't up to date, it may pass CI even though when merged into master it wouldn't.